modules/thermal_hydraulics/test/tests/auxkernels/sound_speed/tests

```diff
@@ -1,7 +1,12 @@
 [Tests]
+  # see relap-7#636
+  # r26174
```

Comment:
typo?

Reply:
no this is copied from the commit.

Reply:
I just don't know what "r26174" is? Is the "r" supposed to be a "#"?

Reply:
Well me neither. And I doubt at the time our issues were is the 26000s. I ll investigate maybe they also mistyped the number

Reply:
came from this
```
    Adding pressure based stabilization for single phase
    
    Author: David Andrs <david.andrs@inl.gov>
    Signed-off-by: Richard Martineau <richard.martineau@inl.gov>
    
    r26174
```

Reply:
Since its meaning has been lost

Reply:
works for me, it s consistent with the approach for the others